lib/wallaby/driver.ex

```diff
@@ -223,10 +256,11 @@ defmodule Wallaby.Driver do
     %{using: "css selector", value: css_selector}
   end
 
-  defp request(method, url, params \\ %{}) do
+  defp request(method, url, params \\ %{}, opts \\ []) do
     headers = [{"Content-Type", "text/json"}]
     body = case params do
       params when map_size(params) == 0 -> ""
+      params when [{:encode_json, false}] == opts -> params
```

Comment:
Thats correct. You can't execute arbitrary functions inside a guard. This might be a good place to use the `with` syntax but I'm leaning towards just moving the whole case statement into its own function that we can then pattern match against.